### PR TITLE
Add Selection and SelectionCard

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Add a Selection and Selection.Card components that are Ant Design Form compatible

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1970** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1970" target="_blank">Graphite</a> 
    * **PR #1971** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1971" target="_blank">Graphite</a> 
      * **PR #1972** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1972" target="_blank">Graphite</a> 
        * **PR #1973** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1973" target="_blank">Graphite</a> 
          * **PR #1974** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1974" target="_blank">Graphite</a> 
            * **PR #1979** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1979" target="_blank">Graphite</a> 
              * **PR #1985** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/1985" target="_blank">Graphite</a> 
                * **PR #2004** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2004" target="_blank">Graphite</a> 
                  * **PR #2005** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2005" target="_blank">Graphite</a> 
                    * **PR #2006** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2006" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2006?utm_source=stack-comment).

File: src/components/Create/components/Selection/SelectionCard.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    if (isSelected) {
+      setSelection?.(undefined)
+      return
+    }
+    setSelection?.(name)
+  }, [isSelected, name, setSelection])
+
+  return (
+    <Border isSelected={isSelected}>
+      <div
```
Comment (tomquirk) [inline src/components/Create/components/Selection/SelectionCard.tsx:60]: **suggestion**: 

```suggestion
      <div
        role="button"
```

File: src/components/Create/components/Selection/SelectionCard.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        style={{ padding: '1.75rem 0', userSelect: 'none' }}
```
Comment (tomquirk) [inline src/components/Create/components/Selection/SelectionCard.tsx:61]: **praise** good call with `userSelect`, makes sense

File: src/components/Create/components/Selection/SelectionCard.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        onClick={onClick}
+      >
```
Comment (tomquirk) [inline src/components/Create/components/Selection/SelectionCard.tsx:63]: **suggestion** WDYT about adding a hover state to cards? Something like adding the `clickable-border` class is prolly fine for now.

File: src/components/Create/components/Selection/SelectionCard.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        <div style={{ padding: '0 1rem' }}>
+          <Row>
+            <Col span={3}>{icon && <RadialBackgroundIcon icon={icon} />}</Col>
```
Comment (tomquirk) [inline src/components/Create/components/Selection/SelectionCard.tsx:66]: **praise** Yeah nice, I back the grid here 👍

**Review by tomquirk — APPROVED** [review]

**Pull request merged**